app/src/main/java/com/android/feedme/ui/profile/ProfileScreen.kt

```diff
@@ -146,12 +153,14 @@ fun ProfileBox(
     padding: PaddingValues,
     profile: Profile?,
     recipeList: List<Recipe>,
+    commentList: List<Comment>,
     navigationActions: NavigationActions,
     profileViewModel: ProfileViewModel,
     recipeViewModel: RecipeViewModel
 ) { // TODO add font
 
   val tabList = listOf("Recipes", "Comments")
+  var selectedTabIndex by remember { mutableStateOf(0) }
```

Comment:
Need quick fix here,
sonar cloud issue : 
Prefer `mutableIntStateOf` instead of `mutableStateOf`
More info [here](https://sonarcloud.io/project/issues?resolved=false&sinceLeakPeriod=true&pullRequest=260&id=feedme-swent-epfl_feedme-android&open=AY_DmtfXh0UU761bfblm)

Reply:
Noted, I'll change it (: